pkg/events/events_test.go

```diff
@@ -0,0 +1,35 @@
+package events
+
+import (
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"testing"
+)
+
+func TestEventMarshalling(t *testing.T) {
```

Comment:
This test is actually testing 3 things: marshal, unmarshal and EventFromJson. Better to split it up in different tests or by using `t.Run(..)`

Also when using assert.NoError, use it in an if statement otherwise a panic can come next.